<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QWidget" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ca">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Mật khẩu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_mk">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tình trạng</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_tt">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Họ và tên</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_hvt">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>CMND</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_cmnd">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Giới tính</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_gt">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Ngày sinh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_ns">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Nghề nghiệp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_nn">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_em">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="td_tt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_hvt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_cmnd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_gt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_ns">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_nn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_em">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="vt_form">
    <item>
     <widget class="QCheckBox" name="ip_vt_reader">
      <property name="text">
       <string>Reader</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ip_vt_manager">
      <property name="text">
       <string>Manager</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ip_vt_librarian">
      <property name="text">
       <string>Librarian</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok_vt">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_vt">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="td_vt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Vai trò</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="op_vt_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="op_vt_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="op_vt_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="td_mk">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thay đổi</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tên đăng nhập</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="op_tdn">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mk_form">
    <item>
     <widget class="QLineEdit" name="ip_mk"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_mk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_mk">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tt_form">
    <item>
     <widget class="QLineEdit" name="ip_tt"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_tt">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_tt">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hvt_form">
    <item>
     <widget class="QLineEdit" name="ip_hvt"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_hvt">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_hvt">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="cmnd_form">
    <item>
     <widget class="QLineEdit" name="ip_cmnd"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_cmnd">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_cmnd">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="gt_form">
    <item>
     <widget class="QRadioButton" name="ip_gt_nam">
      <property name="text">
       <string>Nam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ip_gt_nu">
      <property name="text">
       <string>Nữ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok_gt">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_gt">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ns_form">
    <item>
     <widget class="QSpinBox" name="ip_ns_ngay"/>
    </item>
    <item>
     <widget class="QSpinBox" name="ip_ns_thang"/>
    </item>
    <item>
     <widget class="QSpinBox" name="ip_ns_nam"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_ns">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_ns">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="nn_form">
    <item>
     <widget class="QLineEdit" name="ip_nn"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_nn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_nn">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="em_form">
    <item>
     <widget class="QLineEdit" name="ip_em"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_em">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_em">
      <property name="text">
       <string>CA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
